test/lolex-test.js

```diff
@@ -1560,6 +1560,20 @@ describe("lolex", function () {
             assert.same(Date.clock, this.clock);
         });
 
+        it("takes an object parameter", function () {
+            this.clock = lolex.install({});
+        });
+
+        it("throws a TypeError on a number parameter", function () {
```

Comment:
this should be rewritten to use an assertion.

[see line 210](https://github.com/Peer5/lolex/blob/966e206c88a72c36de9561262c2f64053a2c2300/test/lolex-test.js#L210) for an example